src/Moryx.Model/DbContextManager.cs

```diff
@@ -65,6 +65,16 @@ public DbContextManager(IConfigManager configManager, ILoggerFactory loggerFacto
             }
         }
 
+        private Type DefaultConfigurator()
+        {
+            var sqliteModelConfigurator = ReflectionTool.GetAssemblies()
+                .FirstOrDefault(x => x.GetName().Name.Contains("Moryx.Model.Sqlite"))
+                .GetTypes()
```

Comment:
FirstOrDefault can return `null`, i.e. you need to put a `?` before the next function call and account for `sqliteModelConfigurator` being null in the following 😊